Solutions/PaloAlto-PAN-OS/Playbooks/PaloAlto-PAN-OS-BlockIP/azuredeploy.json

```diff
@@ -1,1047 +1,1068 @@
+        "releaseNotes": [
+        {
+            "version": "1.0.0",
+          "title": "PaloAlto-PAN-OS-BlockIP",
+          "notes": [
+          "Initial version"
+          ]
+        }
+        ]
     },
-    "Teams GroupId": {
-      "defaultValue": "TeamgroupId",
-      "type": "String",
-      "metadata": {
-        "description": "GroupId of the Team channel"
+    "parameters": {
+      "PlaybookName": {
+        "defaultValue": "PaloAlto-PAN-OS-BlockIP",
+        "type": "String",
+        "metadata": {
+          "description": "Name of the Logic App/Playbook"
+        }
+      },
+      "Teams GroupId": {
+        "defaultValue": "TeamgroupId",
+        "type": "String",
+        "metadata": {
+          "description": "GroupId of the Team channel"
+        }
+      },
+      "Teams ChannelId": {
+        "defaultValue": "TeamChannelId",
+        "type": "String",
+        "metadata": {
+          "description": "Team ChannelId"
+        }
+      },
+      "Address Group": {
+        "defaultValue": "AddressGroup",
+        "type": "String",
+        "metadata": {
+          "description": "Address Group"
+        }
       }
     },
-    "Teams ChannelId": {
-      "defaultValue": "TeamChannelId",
-      "type": "String",
-      "metadata": {
-        "description": "Team ChannelId"
-      }
+    "variables": {
+      "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('PlaybookName'))]",
+      "TeamsConnectionName": "[concat('teamsconnector-', parameters('PlaybookName'))]",
+      "PaloAltoConnectorConnectionName": "[concat('PaloAltoConnector-', parameters('PlaybookName'))]"
     },
-    "Address Group": {
-      "defaultValue": "AddressGroup",
-      "type": "String",
-      "metadata": {
-        "description": "Address Group"
-      }
-    }
-  },
-  "variables": {
-    "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('Playbook Name'))]",
-    "TeamsConnectionName": "[concat('teamsconnector-', parameters('Playbook Name'))]",
-    "PaloAltoConnectorConnectionName": "[concat('PaloAltoConnector-', parameters('Playbook Name'))]"
-  },
-  "resources": [
-    {
-      "type": "Microsoft.Web/connections",
-      "apiVersion": "2016-06-01",
-      "name": "[variables('PaloAltoConnectorConnectionName')]",
-      "location": "[resourceGroup().location]",
-	  
-      "properties": {
-        "customParameterValues": {},
-        "api": {
-          "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Web/customApis/PaloAltoConnector')]"
```

Comment:
@v-amolpatil please read the custom connector name from parameter instead of hardcoding at both the places in this as well as all the files